packages/bigtable/src/table.js

```diff
@@ -769,34 +760,24 @@ Table.prototype.insert = function(entries, callback) {
  *     deleted.
  * @param {function} callback - The callback function.
  * @param {?error} callback.err - An error returned while making this request.
- * @param {object[]} callback.mutationErrors - A status object for each failed
- *     mutation.
+ * @param {object[]} callback.err.errors - If present, these represent partial
+ *     failures. It's possible for part of your request to be completed
+ *     successfully, while the other part was not.
  *
  * @example
  * //-
  * // Insert entities. See {module:bigtable/table#insert}
  * //-
- * var callback = function(err, mutationErrors) {
+ * var callback = function(err) {
  *   if (err) {
```

Comment:
Should we document this consistently across the board? For bigquery we aren't checking if err exists, here we do the err + partial error checks in different blocks and for vision we do it in a single block.